documentation/user-interface/main-menu.md

```diff
@@ -71,6 +72,6 @@ The **Window** menu is available on MacOS and provides the following commands:
 The **Help** menu provides the following commands:
 
 * **Welcome** - Displays the **Welcome** page.
-* **Follow TestCafe Studio on Twitter** - Opens the [TestCafe Studio's page](https://twitter.com/TestCafeStudio) on the Twitter website.
-* **About** *(available on Windows)* - Provides TestCafe Studio's version and copyright.
+* **Follow TestCafe Studio on Twitter** - Opens [TestCafe Studio's page](https://twitter.com/TestCafeStudio) on Twitter.
+* **About** *(available on Windows)* - Provides TestCafe Studio's version and copyright information.
 * **Search** *(available on MacOS)* - Allows you to search for the main menu's commands.
```

Comment:
search for commands?

Reply:
Yes, you can type a keyword, and the input will show the menu's commands (that we describe in this topic) that match to this phrase . For example, you can type "Hide" and the Search box will show you the "Hide", "Hide all" commands.

Reply:
Ok